src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTO.kt

```diff
@@ -61,3 +61,8 @@ data class DraftReferralDTO(
     }
   }
 }
+
+data class SelectedDesiredOutcomesDTO(
```

Comment:
please can this go in it's own file

Reply:
it would be fine here, except that it's also used on its own outside of the context of DraftReferralDTO, which ReferralComplexityLevel is not

Reply:
Ok, I did originally, but noticed that InterventionDTO had a DTO in the same file.
I'll do that change.

Reply:
fixed